core/src/utilities/helpers/usersetting-dialog-helpers.js

```diff
@@ -6,7 +6,8 @@ class UserSettingsHelperClass {
 
     processUserSettingGroups() {
         const userSettingGroups = [];
-        const userSettingsSchema = LuigiConfig.getConfigValue('settings.userSettings.userSettingGroups');
+        let userSettingsGroupsFromConfig = LuigiConfig.getConfigValue('userSettings.userSettingGroups');
+        const userSettingsSchema = userSettingsGroupsFromConfig ? userSettingsGroupsFromConfig : LuigiConfig.getConfigValue('settings.userSettings.userSettingGroups');
```

Comment:
Could you also please declare a variable for `LuigiConfig.getConfigValue('settings.userSettings.userSettingGroups')`?   Same as `userSettingsGroupsFromConfig`. 
That will be helpful to understand at first glance.

Reply:
👍